physx.js

```diff
@@ -1671,15 +1701,17 @@ const physxWorker = (() => {
     quaternion.toArray(q)
     size.toArray(s)
 
-    moduleInstance._addBoxGeometryPhysics(
+    const body = moduleInstance._addBoxGeometryPhysics(

```

Comment:
IMO it's best to remove this return value.

See https://github.com/webaverse/app/pull/2731/files#r854723719

Reply:
Removed https://github.com/webaverse/app/pull/2731#discussion_r855127329